From 09a7c209f2113662440cf81b8ea8a7fc2e43cdba Mon Sep 17 00:00:00 2001
From: Jason Chen <b02280@freescale.com>
Date: Wed, 27 Jul 2011 14:06:18 +0800
Subject: [PATCH 0291/2463] ENGR00153757 mxc_hdmi: fix build error of
 mxc_hdmi.c

Fix build error of below:
`mxc_hdmi_remove' referenced in section
`.data' of drivers/built-in.o: defined in discarded section
`.exit.text' of drivers/built-in.o`

Signed-off-by: Jason Chen <b02280@freescale.com>
---
 drivers/video/mxc_hdmi.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/video/mxc_hdmi.c b/drivers/video/mxc_hdmi.c
index 35dec91..383e738 100644
--- a/drivers/video/mxc_hdmi.c
+++ b/drivers/video/mxc_hdmi.c
@@ -2599,7 +2599,7 @@ ealloc:
 	return ret;
 }
 
-static int __exit mxc_hdmi_remove(struct platform_device *pdev)
+static int  mxc_hdmi_remove(struct platform_device *pdev)
 {
 	struct fsl_mxc_lcd_platform_data *pdata = pdev->dev.platform_data;
 	struct mxc_hdmi *hdmi = platform_get_drvdata(pdev);
-- 
1.7.7.4

